english/Journey-to-hell/StoryOfHell.md

```diff
@@ -0,0 +1,26 @@
+# Want to know where the hell is ??
+
+* Visit [here](www.dtu.ac.in)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.